docs/.vuepress/config.js

```diff
@@ -45,7 +45,11 @@ module.exports = {
     ['@simonbrunel/vuepress-plugin-versions', {
       filters: {
         suffix: (tag) => tag ? ` (${tag})` : '',
-        title: (v, vars) => window.location.href.includes('master') ? 'Development (master)' : v + (vars.tag ? ` (${tag})` : ''),
+        title: (v, vars) => {
+          return window.location.href.includes('master') ? 'Development version' :
+                 window.location.href.includes('latest') ? 'Latest version' :
+                 v + (vars.tag ? ` (${tag})` : '') + ' (consider checking the latest version)';
```

Comment:
I think `(outdated)` is a great option! I just thought it would be misleading to have "outdated" next to the latest numbered version. However, @LeeLenaleee's [improvement](https://github.com/chartjs/Chart.js/pull/10446#pullrequestreview-1022188105) here resolves this concern. Will truncate!